src/atomWithStore.ts

```diff
@@ -16,7 +16,7 @@ export function atomWithStore<State, A extends Action = AnyAction>(
   const derivedAtom = atom(
     (get) => get(baseAtom),
     (_get, _set, action: A) => {
-      store.dispatch(action)
+      return store.dispatch(action)
```

Comment:
Looks good! Can we make it an arrow function? Just for a preference.

Reply:
Sure!

Reply:
What could use arrow function?

Reply:
I mean this:
```ts
const derivedAtom = atom(
  (get) => get(baseAtom),
  (_get, _set, action: A) => store.dispatch(action)
)
```